lib/medtronic/processData.js

```diff
@@ -139,24 +165,39 @@ var init = function(config, settings) {
   cfg.settings = _.clone(settings);
 };
 
+var updateConfig = function(config) {
```

Comment:
Hmm, good point. IIRC I created a separate `cfg` object that I could store to a file as part of the binary blob. Now that this functionality is part of `Upload.js`, it should be fine to remove and simplify the code somewhat.

Update: A lot trickier than I thought, haha. Turns out I defined `settings` twice, which took a while to figure out. Changes in commit 75a5328